custom-video-player/script.js

```diff
@@ -4,6 +4,8 @@
 const progress = document.getElementById('progress');
 const timestamp = document.getElementById('timestamp');
 
+var a = foo
```

Comment:
Codacy has a fix for the issue: [Unexpected var, use let or const instead.](https://app.codacy.com/gh/fjrdomingues/vanillawebprojects/pullRequest?prid=9164958)
```suggestion
let a = foo
```